### DISCUSSION TITLE
Customizing. Using Tailwind

### DISCUSSION THREAD

**Discussion by mickdelaney** [dgonz64/react-hook-form-auto#22] [Q&A]
Hi,
I'm looking at whether I could use Tailwind with rhfa. 
The key point about Tailwind being that we'd need to extend the HTML rather than in css.
Could a 'render-props' approach be possible for example ? 
https://github.com/dgonz64/react-hook-form-auto#extending
Or is creating custom css classes _required_ ? 
Regards

**Comment by dgonz64** [comment]
Hello and thanks for the suggesition!

Perhaps the documentation is not particularly clear in this area and I'm sorry, I will try to clarify it. The styles are a list of classes and that probably is not what you need. On the other hand you can override full components in a 'reder-props' way, having the render-props code in the skin. This is covered [here](https://github.com/dgonz64/react-hook-form-auto#overriding-skin), with a tutorial [here](https://github.com/dgonz64/rhfa-playground).

If making a skin is too much, then I need some assistance as to how to ease it. Would those classes be defined in schema, per component or both?
